stylesheets/_component.repeater.scss

```diff
@@ -0,0 +1,5 @@
+.repeater__group {
```

Comment:
If this is the only style, would it be better to use a utility class?

Reply:
Could you give an example of this one, I'm not 100% sure of the usage of utility class here

Reply:
you could either use .u-display-none in the markup, or extend %u-display-none here if you really want the repeater__group class to exist, just felt a bit superfluous

Reply:
I had a go at doing this. I'm not sure `u-display-none` is a viable option in this instance as it is immutable – I need to be able to toggle the visibility of this group. I could switch from toggling the display property to toggling the `u-display-none` class, however I'm weary of coupling the two together.